integration-test/playwright/broker-protection.spec.js

```diff
@@ -1,7 +1,114 @@
+            update: false
+        })
+
+        const data = {
+            firstName: 'shane',
+            lastName: 'osbourne',
+            middleName: 'Alan',
+            city: 'Brooklyn',
+            state: 'NY',
+            addresses: [
+                {
+                    addressLine1: '123 Fake St',
+                    city: 'Brooklyn',
+                    state: 'NY',
+                    zip: '12345'
+                }
+            ]
+        }
+
+        await page.goto('https://www.beenverified.com/app/optout/search')
+        await page.getByRole('button', { name: 'I Agree' }).click()
+        await dbp.simulateSubscriptionMessage('onActionReceived', {
+            state: {
+                action: {
+                    actionType: 'fillForm',
+                    id: '3',
+                    selector: '#search-person',
+                    elements: [
+                        {
+                            type: 'firstName',
+                            selector: '[name=fname]'
+                        },
+                        {
+                            type: 'lastName',
+                            selector: '[name=ln]'
+                        }
+                    ]
+                },
+                data
+            }
+        })
+
+        await dbp.simulateSubscriptionMessage('onActionReceived', {
+            state: {
+                action: {
+                    actionType: 'click',
+                    id: '4',
+                    elements: [
+                        {
+                            type: 'button',
+                            selector: '#optout-search'
+                        }
+                    ]
+                }
+            }
+        })
+
+        const tryClick = {
+            state: {
+                action: {
+                    actionType: 'click',
+                    id: '5',
+                    retry: {
+                        environment: 'web',
+                        strategy: 'fixed',
+                        maxAttempts: 10,
+                        interval: { ms: 1000 }
+                    },
+                    elements: [
+                        {
+                            type: 'button',
+                            selector: '.',
+                            parent: {
+                                profileMatch: {
+                                    selector: '.automation-person-result-data-card',
+                                    profile: {
+                                        name: {
+                                            selector: '.person-name',
+                                            beforeText: ','
+                                        },
+                                        addressCityState: {
+                                            selector: '.person-city'
+                                        }
+                                    }
+                                }
+                            }
```

Comment:
@brianhall everything under 'profile' here (line 87) would work exactly as the matches works in the scan phase.